Morphic/Morphic.pier

```diff
@@ -323,21 +330,21 @@ all crossMorphs answer ==true== to the ==handlesMouseDown:== message.
 
 @@todo Add this method to ==CrossMorph==:
 
-[[[caption=Declaring that ==CrossMorph== will react to mouse clicks.
-CrossMorph>>>handlesMouseDown: anEvent
-	^true
+[[[language=smalltalk|caption=Declaring that ==CrossMorph== will react to mouse clicks|label=scr:mouseEvent
+CrossMorph>>#handlesMouseDown: anEvent
```

Comment:
We can close issue https://github.com/SquareBracketAssociates/UpdatedPharoByExample/issues/85 with this PR.